src/orquestra/sdk/_base/_api.py

```diff
@@ -429,6 +430,8 @@ def wait_until_finished(self, frequency: float = 0.25) -> State:
         while status == State.RUNNING or status == State.WAITING:
             time.sleep(1.0 / frequency)
             status = self.get_status()
+            if print_status:
```

Comment:
Thought: This is kinda missleading. It only prints status when its not ready (failed or succeeded).

Reply:
Ohhh nice catch! That's actually my prototyping code I forgot to roll back.